### PR TITLE
Fix timeouts for cluster boot

### PR DESCRIPTION
**Pull request by elbuo8**
By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

> Describe the purpose of this PR along with any background information and the impacts of the proposed change. For the benefit of the community, please do not assume prior context.
>
> Provide details that support your chosen implementation, including: breaking changes, alternatives considered, changes to the API, etc.
>
> If the UI is being changed, please provide screenshots.

### References

> Include any links supporting this change such as a:
>
> - GitHub Issue/PR number addressed or fixed
> - Auth0 Community post
> - StackOverflow post
> - Support forum thread
> - Related pull requests/issues from other repos
>
> If there are no references, simply delete this section.

### Testing

> Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.
>
> Also include details of the environment this PR was developed in (language/platform/browser version).

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used, if not the default branch

### CODE REVIEW

**Review by joseluisdiaz — APPROVED** [review]

File: lib/db.js [hunk 1]
```diff
@@ -54,10 +54,11 @@ class LimitDBRedis extends EventEmitter {
     this.globalTTL = (config.globalTTL || ms('7d')) / 1000;
 
     const redisOptions = {
+      // a low commandTimeout value would likely cause sharded clusters to fail `enableReadyCheck` due to it running `CLUSTER INFO`
+      // which is a slow command. timeouts are being handled by the client#dispatch method.
       enableOfflineQueue: false,
       keyPrefix: config.prefix,
       password: config.password,
-      commandTimeout: config.commandTimeout || DEFAULT_COMMAND_TIMEOUT,
```
Comment (okeefem3) [inline lib/db.js:60]: Is there going to be any timeout on initial connection?

Comment (elbuo8) [reply lib/db.js:60]: If you timeout the initial connection, you fail to establish a proper connection and it will never recover.

Comment (okeefem3) [reply lib/db.js:60]: Cool, just want to make sure we don't hang forever trying to connect 👍

**Review by okeefem3 — APPROVED** [review]

File: lib/db_ping.js [hunk 2]
```diff
@@ -14,6 +15,7 @@ class DBPing extends EventEmitter {
 
     this.redis = redis;
     this.config = {
+      commandTimeout: 125,
```
Comment (javiersigler) [inline lib/db_ping.js:18]: Shouldn't it be configurable?

**Pull request merged**